assets/src/service/api.js

```diff
@@ -1,8 +1,5 @@
-/* global fetch */
+import useFetch from '../hooks/useFetch'
 
-const get = url => {
-  return fetch(url)
-    .then(res => res.json())
-}
-
-export default get
+export const useGradeData = useFetch(`http://localhost:5001/api/v1/courses/${currentCourseId}/grade_distribution`)
```

Comment:
The `currentCourseId` is not passed in here, the application shows `currentCourseId is not defined` error

Reply:
thanks for pointing that out, i don't have access to any of the data so I am unable to test the app before I make a PR :/